tests/common/plugins/pdu_controller/controller_base.py

```diff
@@ -1,56 +1,56 @@
+PDU controllers.
 
-The PSU controller for actually controlling PSUs must be a subclass of the PsuControllerBase class and must
+The PDU controller for actually controlling PDUs must be a subclass of the PduControllerBase class and must
 implement the methods defined in the base class.
 """
 import os
 import sys
 import subprocess
 
 
-class PsuControllerBase():
+class PduControllerBase():
     """
-    @summary: Base class for PSU controller
+    @summary: Base class for PDU controller
 
-    This base class defines the basic interface to be provided by PSU controller.
+    This base class defines the basic interface to be provided by PDU controller.
 
-    The PSU controller for actually controlling PSUs must be a subclass of the PsuControllerBase class and must
+    The PDU controller for actually controlling PDUs must be a subclass of the PduControllerBase class and must
     implement the methods defined in the base class.
     """
     def __init__(self):
         pass
 
-    def turn_on_psu(self, psu_id):
+    def turn_on_outlet(self, outlet):
         """
-        @summary: Turn on power for specified PSU.
+        @summary: Turn on power for specified PDU.
 
-        @param psu_id: PSU ID, it could be integer of string digit. For example: 0 or '1'
+        @param outlet: PDU ID, it could be integer of string digit. For example: 0 or '1'
         @return: Returns True if operation is successful. Otherwise, returns False
         """
         raise NotImplementedError
 
-    def turn_off_psu(self, psu_id):
+    def turn_off_outlet(self, outlet):
         """
-        @summary: Turn off power for specified PSU.
+        @summary: Turn off power for specified PDU.
 
-        @param psu_id: PSU ID, it could be integer of string digit. For example: 0 or '1'
+        @param outlet: PDU ID, it could be integer of string digit. For example: 0 or '1'
         @return: Returns True if operation is successful. Otherwise, returns False
         """
         raise NotImplementedError
 
-    def get_psu_status(self, psu_id=None):
+    def get_outlet_status(self, outlet=None):
         """
-        @summary: Get current power status of PSUs
+        @summary: Get current power status of PDU outlets
 
-        @param psu_id: Optional PSU ID, it could be integer or string digit. If no psu_id is specified, power status of
-                       all PSUs should be returned
+        @param outlet: Optional PDU ID, it could be integer or string digit. If no outlet is specified, power status of
+                       all PDUs should be returned
         @return: Returns a list of dictionaries. For example:
-                     [{"psu_id": 0, "psu_on": True}, {"psu_id": 1, "psu_on": True}]
-                 If getting PSU status failed, an empty list should be returned.
+                     [{"outlet": 0, "pdu_on": True}, {"outlet": 1, "pdu_on": True}]
```

Comment:
s/pdu_on/outlet_on/ ?

Reply:
Thanks Joe. Done in the last commit. Retested reboot power cycle.